spec/collator.html

```diff
@@ -212,7 +212,7 @@ <h1>
           <dd>
             The returned Number represents the result of an implementation-defined locale-sensitive String comparison of _x_ with _y_.
             The result is intended to correspond with a sort order of String values according to the effective locale and collation options of _collator_, and will be negative when _x_ is ordered before _y_, positive when _x_ is ordered after _y_, and zero in all other cases (representing no relative ordering between _x_ and _y_).
-            String values must be interpreted as UTF-16 code unit sequences as described in es2024, <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and a surrogate pair (a code unit in the range 0xD800 to 0xDBFF followed by a code unit in the range 0xDC00 to 0xDFFF) within a string must be interpreted as the corresponding code point.
+            String values must be interpreted as UTF-16 code unit sequences as described in es2025, <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and a surrogate pair (a code unit in the range 0xD800 to 0xDBFF followed by a code unit in the range 0xDC00 to 0xDFFF) within a string must be interpreted as the corresponding code point.
```

Comment:
```suggestion
            String values must be interpreted as UTF-16 code unit sequences as described in ES2025, <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and a surrogate pair (a code unit in the range 0xD800 to 0xDBFF followed by a code unit in the range 0xDC00 to 0xDFFF) within a string must be interpreted as the corresponding code point.
```

Reply:
No, this is correct as-is: https://tc39.es/ecma402/#normative-references
> Throughout this document, the phrase “es2024, _x_” (where x is a sequence of numbers separated by periods) may be used as shorthand for "ECMAScript 2024 Language Specification (ECMA-262 15th Edition, sub clause _x_)". Where _x_ is followed by more such sequences of period-separated numbers, separated from each other by commas, each such sequence is also a shorthand for the corresponding sub clause of ECMA-262.

Reply:
oof, ok - correct per that definition, but not the way it’s usually referred to outside this document.